frontend/src/hooks/useApi.ts

```diff
@@ -84,44 +67,6 @@ export const useApi = (): IPimsAPI => {
     return axios;
   }, [dispatch]);
 
-  const isPidAvailable = useCallback(
```

Comment:
the following two functions are dead code from citz pims.

Reply:
+1 on cleanup!